packages/transformers/vue/src/VueTransformer.js

```diff
@@ -0,0 +1,358 @@
+// @flow strict-local
+import {Transformer} from '@parcel/plugin';
+import nullthrows from 'nullthrows';
+import {md5FromObject} from '@parcel/utils';
+import ThrowableDiagnostic from '@parcel/diagnostic';
+import type {Diagnostic} from '@parcel/diagnostic';
+import type {TransformerResult} from '@parcel/types';
+import SourceMap from '@parcel/source-map';
+import semver from 'semver';
+import {basename, extname} from 'path';
+
+const MODULE_BY_NAME_RE = /\.module\./;
+
+// TODO: flow
```

Comment:
I assume this TODO can be removed?

Reply:
Yes, probably, but most variables are untyped; not sure if that's OK, but I'll remove it anyway.

Reply:
As long as there's no `$FlowFixMe` all over the place it's fine